cocotb/__init__.py

```diff
@@ -165,20 +165,18 @@ def _initialise_testbench(root_name):
         log.info("Seeding Python random module with supplied seed %d" % (RANDOM_SEED))
     random.seed(RANDOM_SEED)
 
-    module_str = os.getenv('MODULE')
-    test_str = os.getenv('TESTCASE')
-    hooks_str = os.getenv('COCOTB_HOOKS', '')
+    # Setup DUT object
+    from cocotb import simulator
 
-    if module_str is None:
-        raise ValueError("Environment variable MODULE, which defines the module(s) to execute, is not defined.")
+    handle = simulator.get_root_handle(root_name)
+    if not handle:
+        raise AttributeError("Can not find Root Handle (%s)" % root_name)
```

Comment:
Probably would just go with `RuntimeError`, there's not really any attribute access happening here.